pkg/bpf/perf.go

```diff
@@ -138,6 +138,35 @@ import (
 const (
 	EventsMapName   = "cilium_events"
 	MAX_POLL_EVENTS = 32
+
+	PERF_TYPE_HARDWARE   = 0
```

Comment:
Don't you prefer `PERF_TYPE_HARDWARE   = iota` and the remaining ones are automatically assigned?